src/cpp/client_spawner/SentinelServerNode.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file sentinel_server_node.h
+ * @brief Responsible for managing workers and handling their messages
+*/
+#pragma once
+
+#include "StarNode.h"
+
+#define DEBUG
+
+using namespace distributed_algorithm_node;
+
+
+namespace client_spawner {
+
+constexpr const char* WORKER_NOTIFICATION = "worker_notification";
+
+class SentinelServerNode: public StarNode {
+
+public:
+    SentinelServerNode(const string &node_id);
+
+    ~SentinelServerNode();
+
+    void storage_message(string &worker_id, string &message);
```

Comment:
Maybe?
```suggestion
    void storage_message(const string &worker_id, const string &message);
```

Please set them `const` if the strings are not supposed to be changed inside the function.